taichi/common/dynamic_loader.cpp

```diff
@@ -8,6 +8,16 @@
 
 namespace taichi {
 
+bool DynamicLoader::check_lib_loaded(const std::string &lib_path) {
+  bool loaded = false;
+#ifdef WIN32
+  loaded = ((HMODULE)GetModuleHandle(lib_path.c_str()) != nullptr);
```

Comment:
```suggestion
  loaded = ((HMODULE)GetModuleHandleA(lib_path.c_str()) != nullptr);
```

Try this, I don't know if we can directly call ANSI version either.